src/tokenize_cleanup.cpp

```diff
@@ -92,6 +92,12 @@ static void mark_selectors_in_property_with_open_paren(chunk_t *open_paren);
 static void mark_attributes_in_property_with_open_paren(chunk_t *open_paren);
 
 
+/**
+ * Mark all chunks within the scope of a decltype() statement
+ */
+static void mark_cpp_decltype(chunk_t *pc);
```

Comment:
same as above

Reply:
Same as previous remark. I ask because you probably have a better idea as to what the new files should be called.

Reply:
I moved this function into a new file, 'flag_decltype.cpp'